### ISSUE TITLE
->isClicked() on disabled submit button incorrectly returns true

### ISSUE THREAD

**Issue by koku** [symfony/symfony#23759]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.3.5

From the [documentation](https://symfony.com/doc/current/reference/forms/types/submit.html#disabled) of the 'disabled' option of a SubmitType:
"If you don't want a user to be able to click a button, you can set the disabled option to true. It will not be possible to submit the form with this button, not even when bypassing the browser and sending a request manually, for example with cURL."

That last part seems to be incorrect. When manually enabling a disabled button in the browser and submitting the form, calling isClicked() after that submit, will return true.

Code to reproduce in a Controller action:
```php
$form = $this->createFormBuilder()
             ->add('someField', TextType::class)
             ->add('save', SubmitType::class, ['disabled' => true])
             ->getForm();

$form->handleRequest($request);
if ($form->isSubmitted() && $form->isValid()) {

    // Returns true as expected 
    var_dump($form->get('save')->isDisabled());

    // Returns true, while expecting false
    var_dump($form->get('save')->isClicked());

    // Returns true, while expecting false 
    var_dump($form->get('save')->isSubmitted());

    die();
}
```

**Comment by dmaicher** [comment]
But in your example the submit button is not disabled server side?

Did you try this?

```php
$form = $this->createFormBuilder()
             ->add('someField', TextType::class)
             ->add('save', SubmitType::class, [
                'disabled' => true, 
             ])
             ->getForm();

**Comment by koku** [comment]
Oops, I forgot it when writing this isolated example. So yes, this also goes wrong with the 'disabled' option set to true on the submit field. Thanks for the reply, @dmaicher

**Comment by HeahDude** [comment]
Fixed in #25927. Thanks for the report @koku!
